File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.MoreEx_PB-JS-Nested-Conditional-Statements/P04_CarToGo.js
function carToGo(input) {
    let budget = Number(input[0]);
    let seasonType = input[1];

    let classType = "";
    let carType = "";
    let carCost = 0.0;


    if (budget > 500) {
        classType = "Luxury class";
        carType = "Jeep";
        carCost = budget * 0.9;
    } else if (budget > 100) {
        classType = "Compact class";
        if (seasonType === "Summer") {
            carType = "Cabrio";
            carCost = budget * 0.45;
        } else {
            carType = "Jeep";
            carCost = budget * 0.8;
        }
    } else {
        classType = "Economy class";
        if (seasonType === "Summer") {
            carType = "Cabrio";
            carCost = budget * 0.35;
        } else {
            carType = "Jeep";
            carCost = budget * 0.65;
        }
    }
    console.log(classType);
    console.log(`${carType} - ${carCost.toFixed(2)}`);
}

carToGo([450,
    "Winter"
])

File: Java_Basic_Jan_2023_SolutionsOfProblems/P03AdditionalProblems_ConditionalStatementsAdvanced/P04_CarToGo.java
package P03AdditionalProblems_ConditionalStatementsAdvanced;

import java.util.Scanner;

public class P04_CarToGo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        String season = scanner.nextLine();
        String carClass = "";
        String carType = "";

        if (budget <= 100) {
            carClass = "Economy class";
            if (season.equals("Summer")) {
                carType = "Cabrio";
                budget = budget * 0.35;
            } else {
                carType = "Jeep";
                budget = budget * 0.65;
            }
        } else if (budget <= 500) {
            carClass = "Compact class";
            if (season.equals("Summer")) {
                carType = "Cabrio";
                budget = budget * 0.45;
            } else {
                carType = "Jeep";
                budget = budget * 0.8;
            }
        } else {
            carClass = "Luxury class";
            carType = "Jeep";
            budget = budget * 0.9;
        }
        System.out.println(carClass);
        System.out.printf("%s - %.2f", carType, budget);
    }
}
